modules/tinymce/src/plugins/autolink/test/ts/browser/AutoLinkPluginTest.ts

```diff
@@ -47,6 +47,17 @@ UnitTest.asynctest('browser.tinymce.plugins.autolink.AutoLinkPluginTest', (succe
     LegacyUnit.equal(typeUrl(editor, 'mailto:user@domain.com'), '<p><a href="mailto:user@domain.com">mailto:user@domain.com</a>&nbsp;</p>');
     LegacyUnit.equal(typeUrl(editor, 'first-last@domain.com'), '<p><a href="mailto:first-last@domain.com">first-last@domain.com</a>&nbsp;</p>');
   });
+  suite.test('TestCase-TBA: AutoLink: Unexpected urls ended with space', (editor) => {
```

Comment:
This is a good candidate for a property test. I can give you a starting point.

Reply:
I'm not sure what that is, but I'd love to find out.

Reply:
I've pushed up a branch `TINY-4773` with some refactors and property tests.

Reply:
BTW - The "TestCase-TBA" strings should be the jira code, so `TINY-4773`

Reply:
This is a good idea. Thank you for suggesting it.